.github/workflows/android_build.yml

```diff
@@ -72,7 +70,7 @@ jobs:
       with:
         timeout_minutes: 6
         max_attempts: 3
-        command: cd mobile && ./ci/mac_start_emulator.sh
+        command: ./mobile/ci/start_android_emulator.sh
```

Comment:
I renamed this file when I was trying to get these jobs to run on Linux, and even though that didn't work, I felt like this new name for the script is better and worth keeping.